### DISCUSSION TITLE
Simulation data , correlation between PK parameters

### DISCUSSION THREAD

**Discussion by Blduv** [metrumresearchgroup/mrgsolve#1192] [Q&A]
Hello everyone,

I have a question regarding data simulation using mrgsolve. My question concerns the correlations that should be set between pharmacokinetic parameters when simulating models. What is the typical range for setting the correlation between each PK parameter? This includes clearance, intercompartmental clearance, compartments 1, 2, and 3, absorption KA, KMABSI, VMABSI, elimination CL, KM, and VMAX.

Thank you in advance for your insights!

**Comment by kylebaron** [comment]
Hi @Blduv  - 
For correlations, we're talking about correlations between subject-level random effects (off diagonal omega elements)? Usually these are estimated from data. Depending on the estimation method used, most models have a limited set of these correlations estimated (for example, correlation between CL and V or between absorption-related parameters). So I wouldn't expect a model to have all correlations betwee all parameters. 

Do you have these estimates? Or are you trying to make some assumptions in your simulation?

Kyle

**Reply by Blduv** [reply]
Thank you, Kyle, for your response. To clarify further, yes indeed, we are discussing the correlations between subject-level random effects (off-diagonal omega elements) in the context of pharmacokinetic (PK) parameter simulations.

I am working with mrgsolve to simulate data, and I do not have existing datasets to estimate these correlations. Specifically, I am interested in setting up initial correlations for simulation purposes for parameters like clearance (CL), intercompartmental clearance, compartments 1, 2, and 3, absorption rate constant (KA), KMABSI, VMABSI, and elimination parameters such as KM and VMAX.

Since I lack real data from which to derive these correlations, I need to know about typical ranges or standard assumptions used for these PK parameter correlations in simulated datasets. What are the common practices or reasonable assumptions for these correlations when one is starting from scratch?

Thank you again for your insights!
